internal/match/regexp.go

```diff
@@ -135,13 +134,31 @@ func NewMultiLRE(list []*LRE) (_ *MultiLRE, err error) {
 			return nil, fmt.Errorf("%s: no leading phrases", sub.File())
 		}
 		for _, p := range phrases {
+			if p[0] == BadWord {
+				return nil, fmt.Errorf("%s: license can be empty text", sub.File())
```

Comment:
the phrasing here threw me. you're speaking of what could happen if this pattern is accepted. it's really not important, but if you're up for it try something like this:

pattern accepts illegal license: cannot be empty
pattern accepts illegal license: cannot be a single wildcard
etc.

Reply:
Done.